web/packages/extension/src/common.ts

```diff
@@ -32,6 +32,35 @@ class CheckboxOption implements OptionElement<boolean> {
     }
 }
 
+class MaxExecutionDuration_NumberInputOption
```

Comment:
To follow naming conventions, I suggest renaming this class to `MaxExecutionDurationOption`.